HISTORY.rst

```diff
@@ -80,6 +80,9 @@ Unreleased Changes
       (`#1261 <https://github.com/natcap/invest/issues/1261>`_)
     * Updated InVEST logo to use new version with registered trademark symbol.
       (`https://naturalcapitalproject.stanford.edu/invest-trademark-and-logo-use-policy`_)
```

Comment:
I think the readthedocs build doesn't like something about this URL. I believe the trailing `_` is meant to be placed on the text that should contain the link, not on the url itself? If you just want the URL printed as-is, I don't think it needs backticks or underscore or parens.